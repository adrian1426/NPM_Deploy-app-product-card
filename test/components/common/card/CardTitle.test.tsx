import React from 'react';
import renderer from 'react-test-renderer';
import CardTitle from '../../../../src/components/common/card/CardTitle';

describe('CardTitle', () => {
  test('should render CardTitle', () => {
    const wrapper = renderer.create(
      <CardTitle>Mi titulo</CardTitle>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});