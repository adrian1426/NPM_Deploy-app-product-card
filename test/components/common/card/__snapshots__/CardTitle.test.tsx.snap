// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle should render CardTitle 1`] = `
<span
  className="productDescription undefined"
>
  Mi titulo
</span>
`;
